packages/requisitions/src/RequestRequisition/DetailView/RequestLineEdit/hooks.tsx

```diff
@@ -10,7 +10,7 @@ import {
 
 export type DraftRequestLine = Omit<
   RequestLineFragment,
-  '__typename' | 'item' | 'itemStats'
+  '__typename' | 'item'
```

Comment:
the `item.stats` was only giving `0` for all elements.. had to use the version coming off the requisition line to get any data

Reply:
Oh that's weird. The item should be current values and the requisition a snapshot from creation - weird if they differed even just after being created. Do you think it's an API bug?